src/main/resources/additional_definitions.json

```diff
@@ -3,5 +3,9 @@
     "type" : "string",
     "enum" : [ "JSON", "JSON without schema" ],
     "default" : "JSON"
+  },
+  "apply_to": {
+    "type" : "string",
+    "enum" : [ "key", "value" ]
```

Comment:
I was thinking if `message key` and `message value` be more specific and descriptive?